CHANGES.md

```diff
@@ -2,8 +2,13 @@
 Changelog
```

Comment:
I fear changing this file from Restructured Text to Markdown breaks the documentation build as the file is in included in the Sphinx build. See <https://postgraas-server.readthedocs.io/en/latest/changes.html>.